pkgs/top-level/all-packages.nix

```diff
@@ -1863,6 +1863,8 @@ with pkgs;
 
   passExtensions = recurseIntoAttrs pass.extensions;
 
+  quake = callPackage ../applications/misc/quake { };
```

Comment:
```suggestion
  phodal-quake = callPackage ../applications/misc/phodal-quake { };
```

Reply:
The quake repo will move to this [public org](https://github.com/inherd), can I just use the name `quake`?

Reply:
It's not forbidden, but can cause a confusion https://repology.org/project/quake/versions

Reply:
Thanks for reply, for avoid ambiguity, I will use org's name as prefix, `inherd-quake`